src/scripts/main.js

```diff
@@ -18,10 +18,27 @@ const food = {
   },
 };
 
+const treeDiv = document.createElement('div');
+
+treeDiv.id = 'tree';
+document.body.appendChild(treeDiv);
+
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (const key in data) {
+    const li = document.createElement('li');
+
+    li.textContent = key;
+
+    if (Object.keys(data[key]).length > 0) {
+      createTree(li, data[key]);
+    }
+    ul.appendChild(li);
+  }
+  element.appendChild(ul);
 }
```

Comment:
It's a good practice to add some error handling in your functions. In this case, you should check if the `element` and `data` parameters are not `null` or `undefined` before trying to use them. This will prevent potential runtime errors.